pyodide_build/out_of_tree/venv.py

```diff
@@ -95,6 +95,7 @@ def get_pip_monkeypatch(venv_bin: Path) -> str:
         ],
         capture_output=True,
         encoding="utf8",
+        check=True,
```

Comment:
We check the result ourselves with `check_result`.
```suggestion
        check=False
```